lms/views/api/sync.py

```diff
@@ -30,7 +28,7 @@ def sync(request):
     grouping_service = request.find_service(name="grouping")
     assignment_service = request.find_service(name="assignment")
     course = request.find_service(name="course").get_by_context_id(
-        context_id=request.parsed_params["context_id"], raise_on_missing=True
+        context_id=request.lti_user.context_id, raise_on_missing=True
```

Comment:
This is not really necessary for this PR and I'm happy to remove it.

Just demonstrating here how we have to send context and resource data back and forward.